Source_Code/Server/ClueLessServer.java

```diff
@@ -377,36 +380,36 @@ public void processMoveRequest( MoveRequest mr )
        /* First check to see if the action request is from the player who holds the
         * active turn status...
         */
-       if( PlayerList.get( CurrentPlayerIndex ).PlayerNumber == mr.PlayerID)
+       if( PlayerList.get( CurrentPlayerIndex ).PlayerName.equals(mr.PlayerName))
        {
           
           // TODO: update player location
           
           // Announce the move to all players
-          sendToAllPlayers( new Notification("Player " + mr.PlayerID 
+          sendToAllPlayers( new Notification("Player " + mr.PlayerName
```

Comment:
Might be worth removing "Player " in the notification string to remain consistent with the other message notifications.